nimbus/src/lib.rs

```diff
@@ -72,6 +72,14 @@ impl NimbusClient {
             .map(|e| e.branch_slug.clone()))
     }
 
+    pub fn get_experiment_branches(&self, slug: String) -> Result<Vec<Branch>> {
+        Ok(self
+            .get_active_experiments()?
+            .iter()
+            .find(|e| e.slug == slug)
+            .map(|e| e.branches.clone()))
```

Comment:
Since the call to `get_active_experiments()` returns an optional value and has the `?`, you may need to unwrap this to get at the `Vec<Branch>` object.